data/phi0588/abo002/__cts__.xml

```diff
@@ -1,8 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:latinLit:phi0588.abo002" xml:lang="lat" groupUrn="urn:cts:latinLit:phi0588">
             <ti:title xml:lang="lat">Themistocles</ti:title>
             <ti:edition urn="urn:cts:latinLit:phi0588.abo002.perseus-lat2" workUrn="urn:cts:latinLit:phi0588.abo002">
-              <ti:label xml:lang="eng">Themistocles, Cornelii Nepotis Vitae</ti:label>
-              <ti:description xml:lang="eng">Nepos, Cornelius, creator; Fleckeisen, Alfred, 1820-1899, editor; Halm, Karl, 1809-1882, original editor</ti:description>
+              <ti:label xml:lang="lat">Themistocles, Cornelii Nepotis Vitae</ti:label>
```

Comment:
Inconsistent with first file, which dropped the "Cornelii Nepotis Vitae"

Reply:
Dang, inconsistent with only two files, normally it takes me up to three!  But thanks I will make these changes and proceed!